lib/manageiq/release/internationalization.rb

```diff
@@ -0,0 +1,199 @@
+module ManageIQ
+  module Release
+    class Internationalization
+
+      attr_reader :branch, :dry_run
+
+      def initialize(branch: nil, dry_run: true)
```

Comment:
If you change this to 

```ruby
def initialize(branch: nil, dry_run:true, **_)
```

...then you don't need the `.slice` method in the bin script.  You can just pass `**opts` instead, and Ruby will ignore the extra keys.